bottlerocket/agents/src/bin/ec2-resource-agent/ec2_provider.rs

```diff
@@ -99,9 +99,21 @@ impl Create for Ec2Creator {
             .await
             .context(&memo, "Error storing uuid in info client")?;
 
+        memo.current_status = "Getting AWS Secret".to_string();
```

Comment:
Kind of a follow-up to my comment in https://github.com/bottlerocket-os/bottlerocket-test-system/pull/684: since the status there is reported with only a capital letter for the first word (e.g. `No test results`), could we do the same for these custom Strings to keep it consistent?